new/resources-anywhere.md

```diff
@@ -0,0 +1,211 @@
+   - Creates a Provider class extending LWRPBase if it does not already exist.
+   - Sets `use_inline_resources` to true if not explicitly set.
+   - Adds `:create` to `allowed_actions` on the resource.
+   - Calls `action :create do ... end` on the Provider class.
+
+NOTE: this does not address `load_current_resource`, as there are other proposals brewing that may address that in other ways.  Users can open up the provider class with `class Provider < Chef::Provider::LWRPBase` if they need to implement this.
+
+### Backwards Compatibility and Deprecation
+
+Backwards compatibility is king.
+
+- All things that currently work in Chef 12 must continue to work in Chef 12.
+- 99.99% of things that currently work in Chef 12, must continue to work in Chef 13.
+- As much as possible should work in 12.X, so that people in Chef 12 have a story when we move to 13 and so people can begin converting now.
+
+#### Accessing `Chef::Resource::File` or `Chef::Provider::File` directly
+
+In Chef 13, core resources are simply moving out of the Resource and Provider namespaces.  However, in Chef 12, there will be resources and providers who extend from them or construct them directly.
+
+In Chef 12, we will create classes in `Chef::Resource` extending from the real Resources which emit a deprecation warning on initialize or subclassing (`class Chef::Resource::File < Chef::Resources::File ...`).
+
+#### Accessing `File` from a resource or provider method
+
+Right now, when you type `File` in any resource or provider it always refers to `Chef::Resource::File` or `Chef::Provider::File`.  In Chef 13, it will refer to `::File`.
+
+The deprecation warnings for `Chef::Resource::File` suffice for Chef 12 deprecation purposes.
+
+Now for the reverse compatibility part: we want `File` to work.  We will allow anyone who declares a resource *outside* of Chef to type `File` normally by including Chef::Resource::TopLevelConstants directly into their resource.  Whenever a resource is created in `Chef::Resource`, we will check if there is a top level constant with the same name, and add it.  Since only resources *outside* of `Chef::Resource` get this included, anything inside `Chef::Resource` will still refer to the original thing.
+
+LWRPs created via `resources/x.rb` in a cookbook are also exempt from `TopLevelConstants`.
+
+#### Name Conflicts
+
+Resources can try to take the same name in recipe DSL (such as when two cookbooks both implement a similar thing).  Fact of life.  Presently when two custom resources try to take the same name, presently they will do one of two unsatisfactory things:
+
+- Raise a confusing "subclassing the wrong class" error on load; or
+- Silently overwrite and merge with one another, with the last one winning
+
+Now, when you are in separate namespaces, the first thing simply won't happen.
+
+We will avoid the second thing by checking whether the method `my_resource` is already defined, and if it is, we
```

Comment:
did the sentence drop off there, or is the next section supposed to be the end of that sentence? If the later, perhaps a `...` or `:`